src/core/Akka.Tests/IO/TcpIntegrationSpec.cs

```diff
@@ -210,14 +226,123 @@ public void BugFix_3021_Tcp_Should_not_drop_large_messages()
                 var serverMsgs = actors.ServerHandler.ReceiveWhile<Tcp.Received>(o =>
                 {
                     return o as Tcp.Received;
-                }, RemainingOrDefault, TimeSpan.FromSeconds(0.5));
+                }, RemainingOrDefault, TimeSpan.FromSeconds(2));
 
                 serverMsgs.Sum(s => s.Data.Count).Should().Be(testData.Count*3);
             });
         }
 
-        class Aye : Tcp.Event { public static readonly Aye Instance = new Aye();}
-        class Yes : Tcp.Event { public static readonly Yes Instance = new Yes();}
+        [Fact]
+        public void When_multiple_concurrent_writing_clients_Should_not_loose_messages()
```

Comment:
Minor typo here - should be "lose," not "loose"

Reply:
Otherwise the test looks good

Reply:
Thanks! Fixed